File: 几个随机小数和大于一/几个随机小数和大于一.dart
// 计算结果 2.7183122 计算耗时 0.474 秒
import 'dart:math';

const LOOP = 10000000;
void main() {
  var random = Random(DateTime.now().millisecondsSinceEpoch);
  var stopwatch = Stopwatch()..start();
  int n = 0;
  for (int _ = 0; _ < LOOP; ++_)
    for (double x = 0.0; x < 1; x += random.nextDouble()) ++n;
  stopwatch.stop();
  print('计算结果 ${n / LOOP} 计算耗时 ${stopwatch.elapsedMilliseconds / 1000} 秒');
}

File: 几个随机小数和大于一/几个随机小数和大于一.ts
// 计算结果 2.7184451 计算耗时 0.2795 秒
const LOOP: number = 10000000;

const start = performance.now();
let n = 0;
for (let _ = 0; _ < LOOP; ++_)
    for (let x = 0.0; x < 1; x += Math.random())
        ++n;
const end = performance.now();

console.log(`计算结果 ${n / LOOP} 计算耗时 ${(end - start) / 1000} 秒`);